File: hw/sisalema/u2/hw15TestCasesJavaScript/BasicOperations_Javascript/controller/BasicOperation.js
export class BasicOperation {
    static addTwoNumbers(addend1, addend2) {
        let sum = addend1 + addend2;
        // Si quieres redondear a dos decimales:
        sum = Math.round((addend1 + addend2) * 100) / 100;
        return sum;
    }
}

File: ws/ona/u2/ws22BasicOperations/BasicOperations/src/ec/edu/espe/basicoperations/controller/BasicOperation.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package ec.edu.espe.basicoperations.controller;

/**
 *
 * @author Leydi Oña, CodeCrafting Engineers
 */
public class BasicOperation {

    public static float addTwoNumbers(float addend1, float addend2) {
        float sum;
        
        
        sum = Math.round((addend1 + addend2) * 100.0f ) / 100.0f;
        return sum;
    }
}
